### DISCUSSION TITLE
Android support

### DISCUSSION THREAD

**Discussion by carvalholeo** [victorbalssa/abacus#50] [Ideas]
Hello everyone!

Before October 10th, I was an Android user and all apps to manage my Firefly III account it's so bad. So, I started to playing with this app and its source code, to make a version for Android (at that day, my Android phone did die and I bought an iPhone as my main device).

So, I want to know what do you think about an Android version of Abacus.

Clearly, it isn't an easy job or question, but it can be an excellent way of bring this excellent app to Android users.

**Comment by victorbalssa** [comment]
Hey @carvalholeo,
Supporting both Android and iOS is a *very* tough job and it will slow our development of new features,

But we can keep this in mind for future improvement.💡

Thank you for your feedback

**Reply by obxjames** [reply]
Couldn't some of the code be used in flutter? It makes a whole lot of sense for projects like this to use a framework like that?

**Reply by victorbalssa** [reply]
Flutter is a good idea but unfortunately it means to recode everything from scratch..

**Reply by obxjames** [reply]
> Flutter is a good idea but unfortunately it means to recode everything from scratch..

Was afraid as much. Still amazing work none the less, I know my mate would be up for doing a flutter version and I'd certainly be up for doing the UI/UX if that helps.

**Reply by victorbalssa** [reply]
Thank you! 
Unfortunately I'm not good enough in Dart to create a production ready project..
If you want to share any ideas about the UI/UX of Abacus that would be awesome 🔥

**Comment by victorbalssa** [comment]
@carvalholeo 
@obxjames 
Working on this right now.
beta version can be downloaded here (0.8.0): https://play.google.com/store/apps/details?id=abacus.fireflyiii.android.app

**Reply by obxjames** [reply]
Amazing pal! I'm off Tuesday Wednesday for projects. I'll grab, more than happy to have a chat on what I can do to help further.
